src/components/ActivityTabs.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+
+import { Tab, Tabs, Box, Typography } from "@mui/material";
+
+import { Roster } from '@respond/app/(main)/EventPage';
+import { Activity, Participant, ResponderStatus } from '@respond/types/activity';
+import { OutputForm, OutputLink, OutputText, OutputTime } from './OutputForm';
+import { getActivityStatus } from '@respond/lib/client/store/activities';
+import { isActive, reduceDemobilized, reduceSignedOut } from '@respond/types/participant';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function CustomTabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ pt: 2 }}>
+          {children}
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+const reduceActive = (count: number, participant: Participant) => {
+  return count + (isActive(participant?.timeline[0].status) ? 1 : 0);
+}
+
+const reduceStandby = (count: number, participant: Participant) => {
+  return count + (participant?.timeline[0].status === ResponderStatus.Standby ? 1 : 0);
+}
+
+const reduceSignedIn = (count: number, participant: Participant) => {
+  return count + (participant?.timeline[0].status === ResponderStatus.SignedIn ? 1 : 0);
+}
+
+const reduceCheckedIn = (count: number, participant: Participant) => {
+  return count + (isActive(participant?.timeline[0].status) ? 1 : 0);
```

Comment:
This is the same as reduceActive. Should it be?

Reply:
That should be `isSignedIn`, good catch